src/status_im/subs.cljs

```diff
@@ -483,8 +483,7 @@
  :log-level/current-log-level
  :<- [:multiaccount]
  (fn [multiaccount]
-   (or (get multiaccount :log-level)
-       config/log-level-status-go)))
+   (get multiaccount :log-level)))
```

Comment:
`multiaccount :log-level` is now always set because it'll always be initialised during login